src/pages/workspace/WorkspaceResetBankAccountModal.js

```diff
@@ -49,4 +52,11 @@ function WorkspaceResetBankAccountModal(props) {
 WorkspaceResetBankAccountModal.displayName = 'WorkspaceResetBankAccountModal';
 WorkspaceResetBankAccountModal.propTypes = propTypes;
 
-export default withLocalize(WorkspaceResetBankAccountModal);
+export default compose(
+    withLocalize,
```

Comment:
I know this is existing HOC but as this is function component, let's replace this with `useLocalize` hook

Reply:
Done. I have updated HOC with the `useLocalize` hook. @situchan